providers/softlayer/src/test/java/org/jclouds/softlayer/features/VirtualGuestApiLiveTest.java

```diff
@@ -133,7 +134,7 @@ public void testSetTagsOnVirtualGuest() throws Exception {
       Set<TagReference> tagReferences = found.getTagReferences();
       assertNotNull(tagReferences);
       for (String tag : tags) {
-         Iterables.contains(tagReferences, tag);
+         assertThat(Iterables.contains(tagReferences, tag)).isTrue();
```

Comment:
Inspecting this further, this test always failed due to `tagReferences` having a different type than `tag`.  I have corrected it.